lib/money_accessor.rb

```diff
@@ -7,21 +7,41 @@ module ClassMethods
     def money_accessor(*columns)
       Array(columns).flatten.each do |name|
         define_method(name) do
-          value = instance_variable_get("@#{name}")
+          value = _variable_get(name)
```

Comment:
Not sure it matters, but could move the `case self` outside the `define_method` to avoid a dynamic check every time the accessor is called.